inspirehep/migrator/models.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Models for Migrator."""
+
+
+import re
+from datetime import datetime
+from zlib import compress, decompress, error
+
+from invenio_db import db
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from .utils import get_collection_from_marcxml
+
+
+class LegacyRecordsMirror(db.Model):
+    __tablename__ = "legacy_records_mirror"
```

Comment:
Remove legacy_records_mirror from ALEMBIC_SKIP_TABLES in config, as you are adding this table to inspirehep, so we want to start managing it in our alembic.